### ISSUE TITLE
What microcontroller(s) will be we using to interface with sensors?  

### ISSUE THREAD

**Issue by J-Alps** [jmej/capstone2022#16]

**Comment by annalisarose** [comment]
I think Teensyduino for the midi compatibility

**Comment by abbymachines** [comment]
i agree that the teensy is probably a good call, unless other folks like working with other microcontrollers! but i have experience with the teensy and figure other people likely do as well.

website for the teensy: https://www.pjrc.com/teensy/

**Comment by annalisarose** [comment]
Jesse mentioned raspberry pis might be better for OSC messaging, I'll have to investigate. We should also draft a plan for what sensors we'll be using (besides pushbuttons). Should we use this issue to track that?

**Comment by annalisarose** [comment]
E.g. I've heard talk of piezos
